agreement_repair/models/repair.py

```diff
@@ -8,4 +8,3 @@ class Repair(models.Model):
     _inherit = "repair.order"
 
     agreement_id = fields.Many2one("agreement", "Agreement")
-    serviceprofile_id = fields.Many2one("agreement.serviceprofile", "Service Profile")
```

Comment:
Add a glue module with this feature, otherwise we're not supporting the same features that were there before the migration